bip-0032.mediawiki

```diff
@@ -211,7 +211,7 @@ Private and public keys must be kept safe as usual. Leaking a private key means
 Somewhat more care must be taken regarding extended keys, as these correspond to an entire (sub)tree of keys.
 
 One weakness that may not be immediately obvious, is that knowledge of a parent extended public key plus any non-hardened private key descending from it is equivalent to knowing the parent extended private key (and thus every private and public key descending from it). This means that extended public keys must be treated more carefully than regular public keys.
-It is also the reason for the existence of hardened keys, and why they are used for the account level in the tree. This way, a leak of account-specific (or below) private key never risks compromising the master or other accounts.
+It is also the reason for the existence of hardened keys, and why they are used for the account level in the tree. This way, a leak of account-specific (or below) private keys never risks compromising the master or other accounts.
```

Comment:
If you switch "key" to the plural, shouldn't the "s" from "risks" be removed?

Reply:
No, I think "risks" would stay as it is, because it's the verb for "leak" not "keys." If you removed what's in the middle:

`This way, a leak ... never risks compromising the master or other accounts.`

Reply:
Ok, makes sense.

Reply:
i forget my private key but i remember 10 keys of it i wrote it wrongly what can i do can anyone tell me